docs/modules/ROOT/pages/features.adoc

```diff
@@ -0,0 +1,338 @@
+= Features
+:url-asciidoctor-sc-image: https://cdn.jsdelivr.net/gh/asciidoctor/asciidoctor/screenshot.png
+:source-language: asciidoc
+
+The main feature is viewing AsciiDoc as HTML5 directly in the browser.
+Other features cover where those AsciiDoc files are located, and how they are rendered.
+
+You can exercise these features in the  <<try-it-yourself>> section of this page.
+
+[#example]
+== View AsciiDoc as HTML5
+
+The screen capture below shows raw AsciiDoc on the left, and rendered HTML5 on the right.
+
+If the extension is inactive, you'll see the raw AsciiDoc.
+If it is active, you'll see the rendered HTML5.
+
+Reproduce this example yourself by following the instructions further below in <<try-it-yourself>>.
+
+image::{url-asciidoctor-sc-image}["Screen capture of two screens, showing raw AsciiDoc and rendered as HTML5"]
+
+== Auto reload
+
+If the content of the page is changed, the rendered view is automatically refreshed.
```

Comment:
That's partially true, the extension will periodically poll for changes and reload/refresh the preview if the content changed.